src/kubernetes/reflector.rs

```diff
@@ -1,1257 +1,110 @@
-//! Watch and cache the remote Kubernetes API resources.
+//! Intercept [`watcher::Event`]'s.
```

Comment:
Just leaving a comment here for other reviewers that you should expand this file. I initially missed it thinking it was a deleted file.